build/run-docker-tests.sh

```diff
@@ -41,3 +39,13 @@ else
   export CYPRESS_FAIL_FAST_PLUGIN=false
 fi
 npm run test:cypress-headless
+
+if [[ "${SKIP_CLEANUP}" != "true" ]]; then
+  echo "===== E2E Cleanup ====="
+  ${DIR}/cluster-clean-up.sh hub
```

Comment:
Why DIR? Everything else looks like it uses ./build/
```suggestion
  ./build/cluster-clean-up.sh hub
```

Reply:
Shoot. Good catch. That's a copy/paste mistake. Fixing...